packages/server/billing/helpers/teamLimitsCheck.ts

```diff
@@ -97,4 +146,15 @@ export const checkTeamsLimit = async (orgId: string, dataLoader: DataLoaderWorke
       updatedAt: now
     })
     .run()
+
+  const billingLeaders = await getBillingLeaders(orgId, dataLoader)
+  const billingLeadersIds = billingLeaders.map((billingLeader) => billingLeader.id)
+
+  // Enable usage stats
+  if (organization.activeDomain) {
+    await enableUsageStats(billingLeadersIds, orgId)
+
+    // Send push notification
```

Comment:
+1 slightly too many comments. If you're just repeating the next function call, you can omit it